_includes/samples/PriceFeeds/ArbitrumPriceConsumer.sol

```diff
@@ -14,58 +14,60 @@ import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV2V3Interface.sol";
  */
 
 contract ArbitrumPriceConsumer {
-	// Identifier of the Sequencer offline flag on the Flags contract
-    AggregatorV2V3Interface internal priceFeed;
-    AggregatorV2V3Interface internal sequencerUptimeFeed;
+  // Identifier of the Sequencer offline flag on the Flags contract
+  AggregatorV2V3Interface internal priceFeed;
+  AggregatorV2V3Interface internal sequencerUptimeFeed;
 
-    /**
-     * Network: Rinkeby
-     * Data Feed: BTC/USD
-     * Data Feed Proxy Address: 0x2431452A0010a43878bF198e170F6319Af6d27F4
-     * Sequencer Uptime Proxy Address: 0x13E99C19833F557672B67C70508061A2E1e54162
-		 * For a list of available sequencer proxy addresses, see:
-		 * https://docs.chain.link/docs/l2-sequencer-flag/
-     */
-    constructor() {
-        priceFeed = AggregatorV2V3Interface(0x2431452A0010a43878bF198e170F6319Af6d27F4);
-        sequencerUptimeFeed = AggregatorV2V3Interface(0x13E99C19833F557672B67C70508061A2E1e54162);
-    }
+  uint256 private constant STALENESS_THRESHOLD = 3600;
```

Comment:
`GRACE_PERIOD_TIME` instead of `STALENESS_THRESHOLD`

Reply:
Done